src/useResizeObserver/useResizeObserver.ts

```diff
@@ -1,17 +1,17 @@
 import { onMounted, onUnmounted, ref, Ref } from 'vue'
 
 export type UseResizeObserverResponse = {
-  observe: (element: HTMLElement | Ref<HTMLElement | undefined>) => void,
-  unobserve: (element: HTMLElement | Ref<HTMLElement | undefined>) => void,
+  observe: (element: Element | Ref<Element | undefined>) => void,
+  unobserve: (element: Element | Ref<Element | undefined>) => void,
   disconnect: () => void,
-  check: (element: HTMLElement | Ref<HTMLElement | undefined>) => void,
+  check: (element: Element | Ref<Element | undefined>) => void,
 }
 
 export type UseResizeObserverCallback = (entries: ResizeObserverEntry[]) => void
 
 export function useResizeObserver(callback: UseResizeObserverCallback): UseResizeObserverResponse {
 
-  let intersectionObserver: ResizeObserver | null = null
+  let resizeObserver: ResizeObserver | null = null
```

Comment:
😅 thanks 😂

Reply:
I only caught it because I did the same thing 😅